<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Subject</class>
 <widget class="QDialog" name="Add_Subject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_spidokPredmetov">
       <property name="text">
        <string>Список предметов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetSpisokPredmetov"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelAddPredmet">
         <property name="text">
          <string>Добавить предмет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAddPredmet"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="styleSheet">
          <string notr="true">#pushButtonCancel {
padding: 5px;
}</string>
         </property>
         <property name="text">
          <string>Отменить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="styleSheet">
          <string notr="true">#pushButtonDelete {
padding: 5px;
}</string>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="styleSheet">
          <string notr="true">#pushButtonAdd {
padding: 5px;
}</string>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
